modules/preview-react/form-field/lib/FormField.tsx

```diff
@@ -1,37 +1,24 @@
 import React from 'react';
 
 import {createContainer, GrowthBehavior} from '@workday/canvas-kit-react/common';
-import {Flex, FlexProps, mergeStyles} from '@workday/canvas-kit-react/layout';
-import {createStencil} from '@workday/canvas-kit-styling';
-import {system} from '@workday/canvas-tokens-web';
+import {FlexProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 
-import {useFormFieldModel, useFormFieldOrientation} from './hooks';
+import {useFormFieldModel} from './hooks';
 import {FormFieldInput} from './FormFieldInput';
 import {FormFieldLabel} from './FormFieldLabel';
 import {FormFieldHint} from './FormFieldHint';
 import {FormFieldContainer} from './FormFieldContainer';
+import {formFieldStencil} from './formFieldStencil';
 
 export interface FormFieldProps extends FlexProps, GrowthBehavior {
+  /**
+   * The direction the child elements should stack
+   * @default vertical
+   */
+  orientation?: 'vertical' | 'horizontal';
   children: React.ReactNode;
 }
 
-const formFieldStencil = createStencil({
-  base: {
-    border: 'none',
-    padding: system.space.zero,
-    margin: `${system.space.zero} ${system.space.zero} ${system.space.x6}`,
-  },
-  modifiers: {
-    grow: {
-      true: {
-        width: '100%',
-        '[data-width="ck-formfield-width"]': {
-          width: '100%',
-        },
-      },
-    },
-  },
-});
```

Comment:
ah did you move this to prevent circular dep?

Reply:
Yes. It is possible to have the transform completely remove this issue. The reference to `formFieldStencil` is removed from the `parentModifier` instances, but the transform doesn't remove the `import` so it is still a circular dependency.

In practice, it might be fine, but is can be difficult to verify.